lib/generators/avo/templates/locales/avo.en.yml

```diff
@@ -47,6 +47,11 @@ en:
         one: person
         other: people
         zero: people
+      project:
```

Comment:
If I could find a way to inject this during the test runtime, we don't need to update all locale files. The good thing about updating the locale is that is a clear example to others.

Reply:
IMO we can keep this only on the `avo.en.yml` and use it as example and as test

Reply:
@Paul-Bob How to avoid the I18n tests failures? Because the CI complains if I don't add to the other ymls.

Reply:
`"i18n-tasks"` should have an configuration to ignore some translation keys

Reply:
I think is the `ignore_missing:` list on `config/i18n-tasks.yml`

Reply:
<img width="1013" alt="image" src="https://github.com/avo-hq/avo/assets/3402/9ebb4ce1-779b-4347-a55a-800c22f07a4d">

Reply:
Adding `  - 'avo.field_translations.project.status'
` to `L122` in `config/i18n-tasks.yml` don't fix it?

Reply:
@Paul-Bob I found a way to inject only for the spec: aee1e60825f511f7911f9f92eb3ed9e59612c757. I hope you approve. It does not have to mess with the locale files.

Reply:
@Paul-Bob Oh, yes, I can do that: ignore_missing property, I just saw you comment. let me know what approach you prefer, the inject or the `ignore_missing`. I will adjust.

Reply:
@Paul-Bob Check again 👍

Reply:
Looks great @djalmaaraujo thank you!

Reply:
@Paul-Bob What are the next steps?

Reply:
@djalmaaraujo we need to review and locally test it before merging. Thank you for the contribution!